src/hearings/containers/request-hearing/hearing-final-confirmation/hearing-final-confirmation.component.ts

```diff
@@ -37,6 +38,12 @@ export class HearingFinalConfirmationComponent implements OnInit, OnDestroy {
       }, () => {
         this.loadingService.unregister(loadingToken);
       });
+    this.hearingStore.pipe(select(fromHearingStore.getHearingValues)).subscribe((hearingValues) => {
+      if (hearingValues) {
```

Comment:
is caseInfo definitely defined if hearingValues is?

Reply:
@connorpgpmcelroy yes it is